docs/faq.md

```diff
@@ -88,3 +88,15 @@ kubectl delete node lima rancher-desktop
 
 **A:** No, but there are plans to add compatibility.
 
+<!-- #1156 -->
+**Q: How do I fix `permission denied` errors when trying to use Docker no WSL? **
```

Comment:
Did you mean `Docker on WSL`?

Reply:
yse :)

Reply:
I got the same problem on wls2, the solution works perfectly，thanks a lot!

Reply:
> 

I got the same problem on wls2, the solution works perfectly，thanks a lot!